bindings/cs/MongoDB.Crypt/CryptOptions.cs

```diff
@@ -14,28 +14,55 @@
  * limitations under the License.
  */
 
+using System.Collections.Generic;
+using System.Linq;
 
 namespace MongoDB.Crypt
 {
     /// <summary>
-    /// Options to configure mongocrypt with
+    /// Options to configure mongocrypt with.
     /// </summary>
     public class CryptOptions
     {
-        public IKmsCredentials KmsCredentials { get; }
+        public Dictionary<KmsType, IKmsCredentials> KmsCredentialsOptions { get; } = new Dictionary<KmsType, IKmsCredentials>();
```

Comment:
I open for another naming here.

Reply:
Do we really need to expose a writeable dictionary? How about `IReadOnlyList<>`?

Reply:
Alternatively, we could not expose a list at all and just expose two public properties: one for local credentials and one for AWS credentials

Reply:
Per offline discussion, if we keep the dictionary: How about `KmsCredentialsMap`?